README.md

```diff
@@ -6,109 +6,24 @@ Every great Discord server needs a great bot. So we coded up our own. It does a
 
 ## Current Commands
 
-```
-cc!createroles
-```
-
-(admin only) Pulls all badges from Codecademy Discuss and creates a role for each one.
-
----
-
-```
-cc!deleteroles
-```
-
-(admin only) Deletes all the roles added from Codecademy Discuss.
-
----
-
-```
-cc!sendcode [username]
-```
-
-Sends a verification code to your Codecademy Discuss email.
-
----
-
-```
-cc!verify [code]
-```
-
-Verifies that the code entered is valid and gives you a role for every badge you have on Discourse.
-
----
-
-```
-cc!stats
-```
-
-Displays basic server statistics (online members, offline members, total members).
-
----
-
-```
-cc!ping
-```
-
-Pong!
-
----
-
-```
-cc!ban [user] [reason]
-```
-
-Bans a user.
-
----
-
-```
-cc!unban [user]
-```
-
-Unbans a user.
-
----
-
-```
-cc!tempban [user] [lengthoftime] [reason]
-```
-
-Temporarily bans a user for a set time period.
-
----
-
-```
-cc!kick [user] [reason]
-```
-
-Kicks a user from the server.
-
----
-
-```
-cc!mute [user] [reason]
-```
-
-Mutes a user by assigning them a _Muted_ role (denies message sending and reacting privileges).
-
----
-
-```
-cc!unmute [user]
-```
-
-Unmutes a user.
-
----
-
-```
-cc!tempmute [user] [lengthoftime] [reason]
-```
-
-Temporarily mutes a user for a set time period.
-
----
+|      Command        |         Arguments       |    Permission       |                      Description                                                                     |
```

Comment:
I separated them by roles. Do you want a clear cut using multiple tables?

Reply:
@Vic-ST, I initially said to combine all the commands under one table since several of the commands overlap in terms of the roles that can use them. We could separate them into different sections by creating tables for certain levels and above? Like **`Admin Only`**, **`Moderator & Above Only`**, **`Super User & Above Only`**, and **`Everyone`**.

Reply:
@Vic-ST, I initially said to combine all the commands into one table since there's a lot of overlap between commands and the roles that can use them. We could create different sections like **`Admin Only`**, **`Moderator & Above Only`**, **`Super User & Above Only`**, and **`Everyone`** to separate them. Thoughts?

Reply:
@Victoria-DR Oh, I must've missed that, apologizes, disregard this then!